README.md

```diff
@@ -5,25 +5,86 @@ Origin Kubernetes
 [![GoDoc](https://godoc.org/github.com/openshift/origin?status.png)](https://godoc.org/github.com/openshift/origin)
 [![Licensed under Apache License version 2.0](https://img.shields.io/github/license/openshift/origin.svg?maxAge=2592000)](https://www.apache.org/licenses/LICENSE-2.0)
 
-This is the core Kubernetes tracking repo for OKD. For more about OKD, see https://github.com/openshift/okd.
+This repo was previously the core Kubernetes tracking repo for
+[OKD](https://github.com/openshift/okd), and where OpenShift's
+`hyperkube` and `openshift-test` binaries were maintained. As of July
+2020, the purpose and maintenance stratety of the repo varies by
+branch.
 
+## Maintenance of `master` and `release-x.x` branches for 4.6 and above
 
-Contributing
-------------
+These branches no longer include the code required to produce
+`hyperkube` binaries, and are limited to maintaining the `openshift-tests`
+binary.  Responsibility for maintaining hyperkube has transitioned to
+the [openshift/kubernetes](https://github.com/openshift/kubernetes)
+repo.
 
-All contributions are welcome - OKD uses the Apache 2 license and does not require any contributor agreement to submit patches.  Please open issues for any bugs or problems you encounter, ask questions on the OpenShift IRC channel (#openshift-dev on freenode), or get involved in the [Kubernetes project](https://github.com/kubernetes/kubernetes) at the container runtime layer.
+Backports and carries against upstream should be proposed to
+`openshift/kubernetes`. If changes merged to `openshift/kubernetes`
+need to land in `origin`, it will be necessary to follow up with a PR
+to `origin` that bumps the vendoring.
 
-See [HACKING.md](https://github.com/openshift/origin/blob/master/HACKING.md) for more details on developing on Origin including how different tests are setup.
+Branch names are correlated across the 2 repositories such that
+changes merged to a given branch in `openshift/kubernetes` should be
+vendored into the same branch in `origin` (e.g. `master` in
+`openshift/kubernetes` is vendored into `master` in `origin`).
```

Comment:
I don't think we want that. We should rather use branches like `origin-4.5-kubernetes-1.18.3`. Master branches and release-4.x are only used to build hyperkube. We should not mix hyperkube and origin.

Reply:
As per offline discussion, I've added a note below this section indicating that a transition to vendoring origin-specific branches is pending. PTAL.